e2e/bitrise.yml

```diff
@@ -181,37 +241,27 @@ workflows:
         is_always_run: true
         inputs:
         - envs:
-        - files:
+        - files: |-
+            BITRISE_XCTESTRUN_FILE_PATH
         - dirs: |-
-            BITRISE_TEST_DIR_PATH
+            BITRISE_TEST_BUNDLE_PATH
         - deploy_dir: $BITRISE_DEPLOY_DIR
         - deployed_files: |-
             BITRISE_TEST_BUNDLE_ZIP_PATH
-            BITRISE_XCTESTRUN_FILE_PATH
         - deployed_dirs:
-    - script:
-        title: Output check
-        is_always_run: true
-        inputs:
-        - content: |-
-            #!/bin/env bash
-            set -e
-            if [[ "$BITRISE_DEPLOY_DIR/testbundle.zip" != "$BITRISE_TEST_BUNDLE_ZIP_PATH" ]] ; then
-              echo "BITRISE_TEST_BUNDLE_ZIP_PATH (\"$BITRISE_TEST_BUNDLE_ZIP_PATH\") should be: \"$BITRISE_DEPLOY_DIR/testbundle.zip\""
-              exit 1
-            fi
-            if [[ "$BITRISE_DEPLOY_DIR/Debug-iphoneos" != "$BITRISE_TEST_DIR_PATH" ]] ; then
-              echo "BITRISE_TEST_DIR_PATH (\"$BITRISE_TEST_DIR_PATH\") should be: \"$BITRISE_DEPLOY_DIR/Debug-iphoneos\""
-              exit 1
-            fi
-            if [[ "$BITRISE_XCTESTRUN_FILE_PATH" =~ "$BITRISE_DEPLOY_DIR/${BITRISE_SCHEME}_iphoneos[0-9]{2}.[0-9]{1,2}-arm64e.xctestrun" ]] ; then
-              echo "BITRISE_XCTESTRUN_FILE_PATH (\"$BITRISE_XCTESTRUN_FILE_PATH\") should be: \"$BITRISE_DEPLOY_DIR/${BITRISE_SCHEME}_iphoneos{{sdk-version}}-arm64e.xctestrun\""
-              exit 1
-            fi
 
   _run_device_tests:
     steps:
-    - virtual-device-testing-for-ios: { }
+    - virtual-device-testing-for-ios:
+        inputs:
+        - test_devices: iphone13pro,15.2,en,portrait
+
+  _run_test_without_building:
+    steps:
+    - git::https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building.git@step-init:
```

Comment:
This `step-init` git branch based step reference can be changed once the `bitrise-step-xcode-test-without-building` goes live, right?

Reply:
yes